instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -524,18 +525,16 @@ def request_overview():
         )
         return response
 
-    def test_rabbitmq_access(self):
-        """
-        Test that RabbitMQ is accessible
-        """
-        response = self.request_overview()
-        self.assertTrue(response.ok)
-        self.assertIn('"protocol":"amqp/ssl"', response.text)
-
+    @responses.activate
     def test_rabbitmq_request(self):
         """
         Test a simple get request with _rabbitmq_request
         """
+        responses.add(
+            method=responses.GET,
+            url='{}/api/overview'.format(settings.RABBITMQ_API_URL),
+            body='{}'
+        )
```

Comment:
The test is intended to verify that _rabbitmq_request forms a request to `{{service}}/api/overview`. Do you think it would be more useful if I added something more identifying to the mocked body?